src/main/webapp/js/common.js

```diff
@@ -138,10 +138,21 @@ var NAME_MAX_LENGTH = 40;
 var INSTITUTION_MAX_LENGTH = 64;
 
 $(document).on('ajaxComplete ready', function() {
-    $('[data-toggle="tooltip"]').tooltip({html: true, container: 'body'});
+    if (isTouchDevice() === false) {
+        $('[data-toggle="tooltip"]').tooltip({html: true, container: 'body'});
+    }
 });
 
 /**
+ * Checks if the current device is touch based device
+ * Reference: https://github.com/Modernizr/Modernizr/blob/master/feature-detects/touchevents.js
+ * @returns true if touch based device, false otherwise
+ */
+function isTouchDevice(){
+    return true === (('ontouchstart' in window) || window.DocumentTouch && document instanceof DocumentTouch);
```

Comment:
Will `('ontouchstart' in window) || (window.DocumentTouch && document instanceof DocumentTouch)` work?